src/celeritas/ext/VecgeomTrackView.hh

```diff
@@ -9,6 +9,7 @@
 
 #include <VecGeom/base/Config.h>
 #include <VecGeom/base/Cuda.h>
+#include <VecGeom/base/Version.h>
```

Comment:
VecGeom 1.2.4 was the first one to include this header. I'm OK with this, but you'll need to update the version in the main CMakeLists.txt .

Reply:
I've tentative set a minimum VecGeom version required, from 1.1.17 to 1.2.4.
Otherwise, I could not figure out a way to use VECGEOM_VERSION without #including that `<VecGeom/base/Version.h>` header.

Reply:
Yes, I think that's the right way. The only other way would be to add additional cmake logic that writes *another* vecgeom version, which is way too much work to support versions of vecgeom that are effectively broken anyway.